lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -9,18 +9,23 @@ export default (accessToken, userID, requestIDs) => {
   const gravityLoader = gravityLoaderWithAuthenticationFactory(gravityAccessTokenLoader)
 
   return {
+    authenticatedPopularArtistsLoader: gravityLoader("artists/popular"),
+    authenticatedSaleArtworksFilterLoader: gravityLoader("filter/sale_artworks"),
```

Comment:
This is beyond the scope of your PR, but do you think specifying these loaders are authenticated in their names is too verbose, given they are defined in the `loaders_with_authentication` directory? Just curious.

Reply:
This is to avoid a possible [clashing scenario](https://github.com/damassi/metaphysics/blob/add-authenticated-sale-artworks-loader/lib/loaders/loaders_without_authentication/gravity.js#L21) where a loader exists as both auth and unauthed